resource-timing/resources/all_resource_types.htm

```diff
@@ -6,6 +6,35 @@
         <link rel="shortcut icon" href="empty.py?favicon">
     </head>
     <body background='blue.png?id=body'>
+        <script>
+            let count = 0;
+            let totalEventSourceCount = 0;
+
+            function perf_observer(list, observer) {
+                const entries = list.getEntries();
+                const eventSourceCount = entries.filter(e => e.name.includes('eventsource')).length;
+
+                if(eventSource > 0) {
```

Comment:
I don't follow, where is `eventSource` defined?

Reply:
fixed